### PR TITLE
:new: :joystick: cohorts: Add ScioXR (@ScioXR)

### PR DESCRIPTION
**Pull request by jwflory**
This commit adds a new team profile to the VR + AR cohort. This builds
on the work done for unicef/inventory-hugo-theme#13 by @Idadelveloper as
part of unicef/inventory#78.

Screenshot of summary card for new team profile.

Screenshot of full team profile.

### CODE REVIEW

**Comment by jwflory** [comment]
Hi @adriandjura, @AndjelaMihajilovski, & @ScioXR team, I am tagging you here for an initial review of the ScioXR team profile to be published on the UNICEF Open Source Inventory site! This is a new feature on the O.S. Inventory to highlight your open source repositories and points of engagement. The goal is to make your efforts more visible and discoverable.

I need your help to verify the following information before this is officially published:

1. Verify that product summary/elevator pitch, social media links, and other company info are correct.
2. Confirm which [UN Sustainable Development Goals](https://sdgs.un.org/goals) align with your open-source product.

You can leave feedback as a comment on this Pull Request or reach out to me over email. If you are GitHub-savvy, you can also leave a proper review to "sign off" on this Pull Request too, by either approving it or requesting changes. To find the review feature, you can follow these screenshots below to start a review:

<details>

**Screenshot of the "Files changed" menu, where the review feature can be found.**

Screenshot of the "Files changed" menu, where the review feature can be found.
</details>

<details>

**Click the _Review changes_ button to add your review.**

Showing the "Review changes" menu in the upper-right corner, where feedback can be left once the menu is opened.
</details>

<details>

**Screenshot of the review interface.**

Screenshot of the review interface.
</details>

Let me know if you have questions!

File: content/cohorts/vr-ar/scioxr.en.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: ScioXR
+weight: 50
+categories: "team"
+tags: ["team"]
+type: profiles
+
+description: Platform for teachers to make tailored learning STEM content content with VR.
```
Comment (adriandjura) [inline content/cohorts/vr-ar/scioxr.en.adoc:8]: There is a double "content" word

Comment (jwflory) [reply content/cohorts/vr-ar/scioxr.en.adoc]: Fixed in commit eb0117ddf14c9db2022f651eaec734d08a3a9df6.

**Comment by jwflory** [comment]
@adriandjura Good catch. Could you please take a final review?

**Comment by adriandjura** [comment]
Everything is correct and we can close and merge the request

**Comment by jwflory** [comment]
Great. Thanks @adriandjura! Merging. 🌊

**Comment by jwflory** [comment]
This profile is now published on the live site: 

* https://unicef.github.io/inventory/cohorts/vr-ar/
* https://unicef.github.io/inventory/cohorts/vr-ar/scioxr/

**Pull request merged**